### DISCUSSION TITLE
Temurin Download Failures

### DISCUSSION THREAD

**Discussion by gavintoddchurch** [actions/virtual-environments#8675] [General questions]
I've been running the oct 22 release and it keeps failing on various java versions. I've seen it fail on 11 & 21 mostly but seen 17 once. It keeps getting errors like this.

Anyone else experiencing this? 
```
    azure-arm.build_image: Get:1 https://packages.adoptium.net/artifactory/deb jammy/main amd64 temurin-11-jdk amd64 11.0.20.1.0+1 [168 MB]
    azure-arm.build_image: Ign:1 https://packages.adoptium.net/artifactory/deb jammy/main amd64 temurin-11-jdk amd64 11.0.20.1.0+1
    azure-arm.build_image: Err:1 https://packages.adoptium.net/artifactory/deb jammy/main amd64 temurin-11-jdk amd64 11.0.20.1.0+1
==> azure-arm.build_image: E: Failed to fetch https://packages.adoptium.net/artifactory/deb/pool/main/t/temurin-11/temurin-11-jdk_11.0.20.1.0%2b1_amd64.deb  Error reading from server. Remote end closed connection [IP: 146.75.31.42 443]
==> azure-arm.build_image: E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
==> azure-arm.build_image: cat: /usr/lib/jvm/temurin-11-jdk-amd64/release: No such file or directory
```

**Comment by mikhailkoliada** [comment]
Hey, yep, we see it too, and I believe it has been reported upstream

**Reply by gavintoddchurch** [reply]
Thank you! I was trying to find where to report that to adoptium but couldn't find where that would happen.

**Comment by javier-pulido** [comment]
Same error here, do we know if is it going to be fixed soon? We are having some workflows stopped ...

**Reply by ilia-shipitsin** [reply]
our own generation is also blocked. if you have an idea how to mitigate, we would like too

**Reply by gavintoddchurch** [reply]
It looks like the downloads from their GitHub repos are working, is it possible to pull from there?
https://github.com/adoptium/temurin21-binaries/releases/download/jdk-21%2B35/OpenJDK21U-jdk_x64_linux_hotspot_21_35.tar.gz

**Reply by gavintoddchurch** [reply]
Looks like the downloads are still failing. Any other ideas on how to mitigate?

**Reply by mikhailkoliada** [reply]
@GavinToddlds no, unfortunately it is impossible to use GitHub releases as it will be outside ubuntu's ecosystem and will break update-java-alternatives and break few other important tools

**Reply by gavintoddchurch** [reply]
Dang... Thanks for the response. Hopefully the original method will start working soon.

**Reply by gavintoddchurch** [reply]
Just had 2 back to back successful runs.

**Reply by gavintoddchurch** [reply]
Now i'm getting stuck on 
```
azure-arm.build_image: Downloading 'http://security.ubuntu.com/ubuntu/pool/main/o/openssl/libssl1.1_1.1.1f-1ubuntu2.19_amd64.deb' to '/tmp/libssl1.1_1.1.1f-1ubuntu2.19_amd64.deb'...
    azure-arm.build_image: Error — Either HTTP response code for 'http://security.ubuntu.com/ubuntu/pool/main/o/openssl/libssl1.1_1.1.1f-1ubuntu2.19_amd64.deb' is wrong - '404' or exit code is not 0 - '0'. Waiting 30 seconds before the next attempt, 19 attempts left
    ````
